src/smt.rs

```diff
@@ -59,6 +59,10 @@ impl SMTLiteral {
     pub fn bv256_zero() -> Self {
         Self::Hex("0".repeat(64))
     }
+
+    pub fn bv256_one() -> Self {
```

Comment:
Because of https://github.com/leonardoalt/yools/pull/94/files#diff-d69d8a2ccda57291cab86c7e7d3793e86cca96f77b617e11c4a110a9cfb42b92R24 you should be able to just use `0` instead of this.

Reply:
what? don't get it
Also the link doesn't seem to point anywhere specific when I click on it

Reply:
True! It's because of
```
impl SMTFormat for usize {
    fn as_smt(&self) -> String {
        format!("#x{:064x}", self)
    }
}
```